src/test/fuzz/string.cpp

```diff
@@ -279,17 +279,12 @@ FUZZ_TARGET(string)
         const int atoi_result = atoi(random_string_1.c_str());
```

Comment:
In accordance with the CI, maybe this can be removed?

Reply:
> In accordance with the CI, maybe this can be removed?

Could do. Keeping for now, just because I'm not sure it doesn't have value.